dss-api.yml

```diff
@@ -2666,6 +2846,54 @@ definitions:
       - type
       - uuid
       - version
+  EventsManifest:
+    type: object
+    properties:
+      journal_id:
+        description: The event journal identifier.
+        type: string
+      from_date:
+        type: string
+        format: DSS_VERSION
+        description: DSS_VERSION format timestamp of the first event in the journal.
+        type: string
+      to_date:
+        type: string
+        format: DSS_VERSION
+        description: DSS_VERSION format timestamp of the last event in the journal.
+        type: string
+      size:
+        type: integer
+        description: Size of the journa in bytes.
+      events:
+        description: A list describing the event content of the journal.
+        type: array
+        items:
+          type: object
+          properties:
+            event_id:
+              description: The event identifier.
+              type: string
+            timestamp:
+              description: DSS_VERSION format timestamp of the event.
+              type: string
+            offset:
+              description: The location of the event data in the journal, as 0-indexed byte offset.
+              type: integer
+            size:
+              description: The size of the event data in bytes
+              type: integer
+          required:
+            - event_id
+            - timestamp
+            - offset
+            - size
+    required:
+      - journal_id
+      - from_date
+      - to_date
+      - size
+      - events
```

Comment:
nit: add `{dss_api:f'https://{os.environ.get("API_DOMAIN_NAME")}'`

Reply:
nit: provide `{"page_count":len(events),"per_page":per_page}` for the users, just a nice to have

Reply:
nit: provide next_link for the user in the response of the body.

Reply:
I don't think this information belongs in the event manifest (which I can describe in greater detail), although it may be beneficial to include it in the paginated response to GET /events.

Reply:
^ i agree, apologies i was writing comments in the wrong space.